R/multiEquatError.r

```diff
@@ -1,19 +1,30 @@
-multiEquatError <- function (x1, x2, x3, difBound = 1, dependentDIF =FALSE, testletStr = NULL, verbose=TRUE ){
-       liste<- list(x1, x2, x3)
-       chk1 <- checkInput(liste)
-       if ( "derived.par" %in% colnames(x1) ) {                                 
-            obj  <- prepareAndCheckEatModelObject(liste, difBound=difBound, verbose=verbose)
-            link <- lapply(names(obj), FUN = function (dim ) { tripleEquatError(e1=obj[[dim]][[1]][,c("item", "est")], e2=obj[[dim]][[2]][,c("item", "est")], e3= obj[[dim]][[3]][,c("item", "est")], dependentDIF=dependentDIF, testletStr=testletStr, difBound=difBound, calledForEatModel = TRUE)})
-            names(link) <- names(obj)
-       } else {                                                                 
-            link <- tripleEquatError(e1=x1, e2=x2, e3=x3, dependentDIF=dependentDIF, testletStr=testletStr, difBound=difBound, calledForEatModel = FALSE)
-       }
-       return(link) }
 
+multiEquatError <- function (x1, x2, x3, difBound = 1, dependentDIF =FALSE, testletStr = NULL, verbose=TRUE){
+  lapply(list(x1, x2, x3), checkmate::assert_data_frame, min.cols = 2)

```

Comment:
I assume this will fail. The object `liste` is no longer created but is needed in line 9. Maybe better:
```r
liste<- list(x1, x2, x3)
lapply(liste, checkmate::assert_data_frame, min.cols = 2)
```